chia/wallet/cat_wallet/cat_wallet.py

```diff
@@ -622,7 +622,7 @@ async def generate_unsigned_spendbundle(
         for coin in cat_coins:
             if first:
                 first = False
-                announcement = Announcement(coin.name(), std_hash(b"".join([c.name() for c in cat_coins])), b"\xca")
+                announcement = Announcement(coin.name(), std_hash(b"".join([c.name() for c in cat_coins])))
```

Comment:
It looks like this comment is now out of date and should be updated?  If so I'll add this to the post-merge quick follow list.

https://github.com/Chia-Network/chia-blockchain/blob/catchup/main_from_long_lived_post_1.5.0_916ccee549b1594349c19fcbb0d0d210698c899b/chia/types/announcement.py#L8-L12